src/Kernel32.Desktop/Kernel32.cs

```diff
@@ -684,5 +684,67 @@ public static partial class Kernel32
         /// </returns>
         [DllImport(nameof(Kernel32), SetLastError = true)]
         public static extern bool IsWow64Process(SafeObjectHandle hProcess, out bool Wow64Process);
+
+        /// <summary>Retrieves the window handle used by the console associated with the calling process.</summary>
+        /// <returns>
+        ///     The return value is a handle to the window used by the console associated with the calling process or
+        ///     <see cref="IntPtr.Zero" /> if there is no such associated console.
+        /// </returns>
+        [DllImport(nameof(Kernel32))]
+        public static extern IntPtr GetConsoleWindow();
```

Comment:
@ffMathy (since we were talking about this on #56) This is an example of an IntPtr that does represent a handle, but shouldn't be closed by the caller. A typical `SafeHandle` would "own" this and want to close it later, which would be a problem.
So we can leave it as `IntPtr` to avoid the problem. However if all other APIs that would want to consume this handle from its caller take `SafeHandle`, that can make it more difficult. In which case, I wonder if we could create a `SafeUnownedHandle` and type it as that. This type's constructor would assume the handle isn't owned so that Close isn't called, yet it would still be typed as a `SafeHandle`-derived class so other APIs could accept it.